spec/services/scheduled_publishing_failed_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+RSpec.describe ScheduledPublishingFailedService do
+  describe "#call" do
+    let(:edition) { create(:edition, :scheduled) }
+
+    it "updates the status of the edition" do
+      expect { ScheduledPublishingFailedService.new.call(edition.id) }
+        .to change { edition.reload.state }
+        .from("scheduled")
+        .to("failed_to_publish")
+    end
+
+    it "maintains the scheduling details from the previous status" do
+      expect { ScheduledPublishingFailedService.new.call(edition.id) }
+        .not_to(change { edition.reload.status.details })
+    end
+
+    context "when an edition is not scheduled" do
```

Comment:
As we discussed previously, I don't think we have a precedent for these kinds of tests, since they are basically just assertions.

Reply:
Will nuke